### PR TITLE
 Test on Jruby

### PR DESCRIPTION
**Pull request by p-mongo**

### CODE REVIEW

**Comment by p-mongo** [comment]
This is also blocked on https://github.com/10gen/buildhost-configuration/pull/107.

**Comment by p-mongo** [comment]
All special OS configurations were removed because they are broken for various reasons and are blocked on build team.

File: .evergreen/.evg.yml [hunk 1]
```diff
@@ -480,18 +480,24 @@ axes:
...
-      - id: "rhelzseries"
-        display_name: "RHEL 7.2 Z-Series"
-        run_on: rhel72-zseries-test
-      - id: "rhelpower"
-        display_name: "RHEL 7.1 PowerPC"
-        run_on: rhel71-power8-test
-      - id: "ubuntu1604-arm64-small"
-        display_name: "Ubuntu 16.04 ARM64"
-        run_on: ubuntu1604-arm64-small
+  #- id: "special-os"
```
Comment (saghm) [inline .evergreen/.evg.yml]: To make sure I'm understanding correct, the intention is that we'll add these back once the build team makes the changes we require? (Ditto for the other commented out sections)

Comment (saghm) [reply .evergreen/.evg.yml]: Awesome, thanks

**Pull request merged**